static/docs/user-guide/dvc-files-and-directories.md

```diff
@@ -4,7 +4,7 @@ Once initialized in a <abbr>project</abbr>, DVC populates its installation
 directory (`.dvc/`) with the internal files and directories needed for DVC
 operation:
 
-### DVC internal files and directories
+### DVC internals
```

Comment:
probably not a good naming here ... I would read this as internals of the DVC as a program, not files ... to be honest "internal" is a pretty bad term here in general ... `.dvc/config` is not internal, it provides a public API. Let's take a look if Git has a similar section or how do people refer to `.git` directory content.

Reply:
`.git/`:
- "contains all of your necessary repository files — a Git repository skeleton" ([Git Basics](https://git-scm.com/book/en/v2/Git-Basics-Getting-a-Git-Repository) official docs)
- "where almost everything that Git stores and manipulates is located" ([Git **Internals**](https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain#ch10-git-internals) official docs)
- "Git directory structure... special folder where all the git stuff is" ([Inside Git: .Git directory](https://githowto.com/git_internals_git_directory))
- "subfolder with all the git related revision history of your repo" ([What is a bare git repository?](http://www.saintsjd.com/2011/01/what-is-a-bare-git-repository/))

`.git/config`:
- "config(uration) file" 
- "contains your project-specific configuration options" ([Git **Internals**](https://git-scm.com/book/en/v2/Git-Internals-Plumbing-and-Porcelain#ch10-git-internals) official docs)

The **Git Internals** chapter linked above desribes itself as "... this is where we’ll go over the inner workings and implementation of Git."

Reply:
so, this should be specifically `Inside DVC: .dvc directory` then? Not more general and broad "DVC internals"?

Reply:
Maybe I'll just remove this H2! It's not really needed is it? But I also need to review the usage of the term "internal" throughout the docs...

Reply:
Sounds good to me!

Reply:
Done. And reviewed term in e17ab1a.